src/components/PromisesTable.vue

```diff
@@ -79,14 +93,48 @@ export default {
     exclude: { type: Array, default: () => [] }
   },
   data: () => ({
-    search: ''
+    search: '',
+    filterStatus : '',
+    statusOptions: [
+    {
+      value: '',
+      label: 'All'
+    }, {
+      value: 'Review Needed',
+      label: 'Review Needed'
+    }, {
+      value: 'Not Started',
+      label: 'Not Started'
+    }, {
+      value: 'In Progress',
+      label: 'In Progress'
+    }, {
+      value: 'Fulfilled',
+      label: 'Fulfilled'
+    }, {
+      value: 'Partially Fulfilled',
+      label: 'Partially Fulfilled'
+    }, {
+      value: 'At Risk',
+      label: 'At Risk'
+    }, {
+      value: 'Retracted',
+      label: 'Retracted'
+    }, {
+      value: 'Broken',
+      label: 'Broken'
+    }],
   }),
   methods: { formatDate },
   computed: {
     filteredPromises () {
-      return this.promises.filter(promise => {
+      var results = this.promises.filter(promise => {
```

Comment:
let's stick to ES6, so declare variables by either using `let` or `const`

Reply:
Kay. No problems